galoy-wallet/src/client/mod.rs

```diff
@@ -0,0 +1,124 @@
+mod error;
+mod queries;
+
+use graphql_client::reqwest::post_graphql;
+use reqwest::Client as ReqwestClient;
+
+use error::*;
+pub use queries::*;
+
+use auth_code::*;
+use btc_price::*;
+use default_wallet::*;
+use user_login::*;
+
+#[derive(Debug, Clone)]
+pub struct GaloyClient {
+    client: ReqwestClient,
+    config: GaloyClientConfig,
+}
+
+#[derive(Debug, Clone)]
+pub struct GaloyClientConfig {
+    pub api: String,
+    pub phone_number: String,
```

Comment:
we probably won't store the phone number but more likely the JWT, specially for the dealer.

Reply:
I will eventually add the JWT, but I have no need for it yet until I can get the `authentication_code()` and `login()` functions to work with my local environment for galoy backend.

Reply:
The current dealer gets phone/code injected and creates its own auth token - so we are using the same model here. It means less manual steps for provisioning Stablesats in a live environment.